### PR TITLE
#41delete comment

### PR DESCRIPTION
**Pull request by IrinaSTA**

### CODE REVIEW

**Comment by IrinaSTA** [comment]
Please don't merge this yet!

**Comment by IrinaSTA** [comment]
Please merge when you've reviewed. Thanks

**Comment by IrinaSTA** [comment]
Can only delete your own - see a message if attempting to delete someone else's
They can see a helpful message that the delete was successful

File: db/schema.rb [hunk 1]
```diff
@@ -15,6 +15,13 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "articles", force: :cascade do |t|
```
Comment (rbbri) [inline db/schema.rb]: @IrinaSTA Could you run a local migration on your mac to remove this, without commiting the migration. This should solve this

**Comment by Caitlin-cooling** [comment]
:shipit:

**Comment by IrinaSTA** [comment]
Since this has been reviewed by Caitlin and Robin, I will merge myself after addressing the comments.

**Pull request merged**